### DISCUSSION TITLE
Who's using

### DISCUSSION THREAD

**Discussion by xvik** [xvik/gradle-use-python-plugin#18] [Show and tell]
If you use the plugin please share how are you using it. For open-source projects would be great to post the link (as a usage example that will hopefully help others)!

**Comment by lupino3** [comment]
The open source CPU simulator [EduMIPS64](https://www.edumips.org) is using this plugin to automate building the Sphinx-based user manual from Gradle.

Before adopting this plugin, we had to manually guess where the Python executable would be, how it would be called across multiple platforms, rely on the user to install the dependencies (Sphinx and rst2pdf) manually.

Adopting the plugin was a very straightforward process (see PR: https://github.com/EduMIPS64/edumips64/pull/550) and it allowed us to greatly simplify our developer workflow and led to an overall better development experience thanks to integrated dependency management and the ability to install dependencies in a virtualenv.

During the adoption, we hit a minor bug in the plugin (https://github.com/xvik/gradle-use-python-plugin/issues/14). The author was very helpful and responsive and fixed it very quickly.

Thanks a million! 👍🏼
